samples/sensor/bmm150/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _bmm150:
+
+BMM150 GeoMagnetic Sensor
+#########################
+
+Overview
+********
+
+Sample application that periodically reads magnetometer (X, Y, Z) data from
```

Comment:
Make this a sentence.  How about starting with, 

     This sample application periodically reads ...